scripts/metadata_table.py

```diff
@@ -348,6 +348,14 @@ def __init__(self, run_env, table_name_in=None, table_type_in=None,
             # end if
             self.__start_context = ParseContext(context=self.__pobj)
             self.__init_from_file(known_ddts, self.__run_env, skip_ddt_check=skip_ddt_check)
+            # Set absolute path (if available) for all dependencies
```

Comment:
should this be something like
```suggestion
            # Set absolute path (including relative path if available) for all dependencies
```